src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -78,13 +78,17 @@ const gridMarginSmall = css`
 `;
 
 const Wrapper = styled(ConstrainedLargeGrid)`
+  ${isMediaContent =>
+    !isMediaContent &&
```

Comment:
Maybe something like `Wrapper = isMapContent ? Grid : styled(Grid)` would be clearer